packages/project/src/db/props.ts

```diff
@@ -73,12 +59,60 @@ export const loadByTreeId = async (treeId: Tree["id"]) => {
       continue;
     }
 
+    props.push(prop);
+  }
+
+  return props;
+};
+
+export const loadByTreeId = async (treeId: Tree["id"]) => {
+  const tree = await prisma.tree.findUnique({
+    where: { id: treeId },
+  });
+
+  if (tree === null) {
+    return [];
+  }
+
+  const props = await parseProps(tree.props);
+
+  const instancePropsMap = new Map<string, InstanceProps>();
+  for (const propsItem of props) {
+    let instanceProps = instancePropsMap.get(propsItem.instanceId);
+    if (instanceProps === undefined) {
+      instanceProps = {
+        id: "",
+        instanceId: propsItem.instanceId,
+        treeId,
+        props: [],
+      };
+      instancePropsMap.set(propsItem.instanceId, instanceProps);
+    }
     instanceProps.props.push(propsItem);
   }
 
   return Array.from(instancePropsMap.values());
 };
 
+export const serializeProps = (props: Props) => {
```

Comment:
Do we need to add tests for serialize/parse functions? It feels like the probability of breaking this stuff in the future is high

Reply:
We have almost same code for styles. Maybe we need to change how we handle assets.